pkg/builtinchecks/yamls/unsafe-proc-mount.yaml

```diff
@@ -0,0 +1,7 @@
+name: "unsafe-proc-mount"
+description: "Alert on deployments with unsafe /proc mount on containers"
+remediation: "Ensure container does not exposes unsafe parts of /proc."
```

Comment:
Say how this could be done (ie, which value in the YAML should be changed). Also a link to documentation/reasoning would be helpful.

Reply:
updated with instruction and a link for details.